indico/modules/events/sessions/templates/display/session_display.html

```diff
@@ -59,14 +59,11 @@
                 {% if g.static_site %}
                     <a href="{{ url_for('sessions.export_ics', sess) }}" class="i-button icon-calendar"></a>
                 {% else %}
-                    <div id="session-calendar-link" data-session-id="{{ sess.id }}"
-                         data-event-id="{{ sess.event_id }}"></div>
+                    <div id="session-calendar-link" data-session-id="{{ sess.id }}" data-event-id="{{ sess.event_id }}"
+                         data-session-contrib-count="{{ sess.contributions|length }}"-></div>
                 {% endif %}
             {%- endif %}
         </div>
-        {% if sess.start_dt -%}
```

Comment:
I think this check may need to stay. Exporting a poster session's contributions to ical doesn't make sense since they have no time.

Reply:
If I'm not mistaken, the same check and the same element is added in lines 58-65:

https://github.com/indico/indico/blob/749bbc98f82251ef874c43e799509e201f67dc88/indico/modules/events/sessions/templates/display/session_display.html#L58-L65

I removed this because the final html ended up with two divs with that id. Am I missing something?

Reply:
Oh right. I didn't notice the `start_dt` check being around the other code as well.